airbyte-integrations/connectors/destination-selectdb/src/main/java/io/airbyte/integrations/destination/selectdb/SelectdbCopyInto.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.selectdb;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Preconditions;
+import io.airbyte.integrations.destination.selectdb.exception.CopyIntoException;
+import io.airbyte.integrations.destination.selectdb.exception.UploadException;
+import io.airbyte.integrations.destination.selectdb.http.HttpPostBuilder;
+import io.airbyte.integrations.destination.selectdb.http.HttpPutBuilder;
+import io.airbyte.integrations.destination.selectdb.utils.ResponseUtils;
+import org.apache.http.Header;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.entity.ByteArrayEntity;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.util.EntityUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.*;
+
+public class SelectdbCopyInto {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SelectdbCopyInto.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final String UPLOAD_URL_PATTERN = "https://%s/copy/upload";
```

Comment:
@catpineapple does SelectDb support https or is it only http?

Reply:
Only http is supported currently. @itaseskii